### DISCUSSION TITLE
your live app demo is not working with method sign_eth

### DISCUSSION THREAD

**Discussion by johnnydepp9** [WalletConnect/walletconnect-monorepo#3163] [Developer Support]
I used your live dApp Demo but i can not sign with method eth_sign.

Live App Demo: https://react-app.walletconnect.com/
Source: https://github.com/WalletConnect/web-examples/tree/main/dapps/react-dapp-v2

Please let me known how to fixed.

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/110464670/6b026a21-8cc2-4b5e-b92b-e901f8b2cd4e)

**Comment by otiszero** [comment]
I have the same issue :(

**Comment by glitch-txs** [comment]
eth_sign is not supported by most wallets, try "personal_sign" instead.
Btw are you developing a wallet?

**Reply by johnnydepp9** [reply]
I am developing a DApp, and use metamask app (in IOS) for connecting

**Reply by glitch-txs** [reply]
Yes, it's disabled by default in MetaMask, this might be useful:

https://support.metamask.io/hc/en-us/articles/14764161421467-What-is-eth-sign-and-why-is-it-a-risk-

https://docs.metamask.io/wallet/concepts/signing-methods/#deprecated-signing-methods
